effect_player.hpp

```diff
@@ -42,7 +41,6 @@ namespace bnb::oep
         bnb::interfaces::pixel_format make_bnb_pixel_format(pixel_buffer_sptr image);
 
     private:
-        bnb::utility m_utility;
```

Comment:
it might be better to do it like this:
static std::unique_ptr<bnb::utility> s_utility {nullptr};